src/App.tsx

```diff
@@ -20,17 +25,111 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    stepChange: 3,
+    frameSizeChange: 3,
+    itemWidthChange: 130,
+    duration: 1000,
+    infiniteImg: false,
+  };
+
+  handleStepChange = (value: number) => {
+    this.setState({ stepChange: value });
+  };
+
+  handleFrameSizeChange = (value: number) => {
+    this.setState({ frameSizeChange: value });
+  };
+
+  handleItemWidthChange = (value: number) => {
+    this.setState({ itemWidthChange: value });
+  };
+
+  handleDurationChange = (value: number) => {
+    this.setState({ duration: value });
+  };
+
+  handleInfiniteChange = (value: boolean) => {
+    this.setState({ infiniteImg: value });
```

Comment:
Combine repeatable code into 1 function
```suggestion
  handleChange = (value: number, key: string) => {
    this.setState({ [key]: value });
  };
```